src/components/MenuBar/MenuBar.style.ts

```diff
@@ -0,0 +1,117 @@
+import styled from 'styled-components';
+
+// Menu Bar container styling
+export const StyledMenuBar = styled.div`
+	display: flex;
+	justify-content: space-between;
+	align-items: center;
+	background-color: #e97462; /* Primary Color */
+	padding: 16px;
+	box-shadow: 0 2px 4px rgba(0, 0, 0, 0.1); /* Subtle shadow */
+
+	nav {
+		display: flex;
+		gap: 24px;
+
+		// Hamburger menu: Hidden on large screens, shown on smaller screens
+		@media (max-width: 768px) {
+			display: none;
+			&.open {
+				display: block;
+			}
+		}
+	}
+`;
+
+// Icon Image styling for the icon.svg from the public folder
+export const IconImage = styled.img`
+	height: 40px;
+	width: 40px;
+	margin-right: 16px; /* Adjust margin as necessary */
+`;
+
+// MenuItem styling with updated secondary color for text
+export const MenuItem = styled.a`
+	font-family: 'Inter Tight', sans-serif; /* Font Family */
```

Comment:
Please use theme values for font family, font size, font weight, padding, and margins throughout. You can find all the available theme values [here in the default theme definition](https://github.com/Redback-Operations/redback-ui/blob/main/src/themes/default.ts).